src/components/dialog/styles/dialog.css

```diff
@@ -0,0 +1,65 @@
+.main {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background: #FFF;
```

Comment:
can you please move colors, fonts, shadows etc.. out to `src/styles/variables.css` file and `@import` it at the top?